docker.json

```diff
@@ -0,0 +1,13 @@
+{
+  "images": {
+    "lighthouse": {
+      "current": "sigp/lighthouse:v1.1.3",
```

Comment:
Can we just hardcode "recommended" version but display all versions from github in dropdown?

Reply:
Fetch from this endpoint https://api.github.com/repos/nodefactoryio/chainguardian/releases (chaneg repo and owner to lighthouse)

Reply:
think it's better to keep it like this is for now and discuss this in the next weekly?